### PR TITLE
revert changes

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2183
revert changes https://github.com/MicrosoftDocs/windows-powershell-docs/commit/de9804871db683335c9e846d18eb920d4cd45ea9#diff-e8fa4151e0a760501051972a6a346a4eb82006ef1308d959ad0998b0704921c7 and https://github.com/MicrosoftDocs/windows-powershell-docs/commit/92773b22ea7bc498e910b1c74f883a8bced06598#diff-e8fa4151e0a760501051972a6a346a4eb82006ef1308d959ad0998b0704921c7

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/failoverclusters/Set-ClusterLog.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Accept wildcard characters: False
 
 ### -Size
 Specifies the log size to set for the cluster.
-The acceptable values for this parameter are:`8` MB to `2048` MB.
+The acceptable values for this parameter are:`8` MB to `1024` MB.
```
Comment (dariomws) [inline docset/windows/failoverclusters/Set-ClusterLog.md:120]:
```suggestion
The acceptable values for this parameter in Windows 2016 are:`8` MB to `1024` MB.
The acceptable values for this parameter in Windows 2019 are:`8` MB to `2048` MB.
```

**Review by dariomws — CHANGES_REQUESTED** [review]
I agree with @carlosmayol.

The difference between 2016 and 2019 must be specified.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**